web/react/components/create_post.jsx

```diff
@@ -176,6 +176,7 @@ export default class CreatePost extends React.Component {
 
         PostStore.storePendingPost(post);
         PostStore.storeDraft(channel.id, null);
+        PostStore.jumpPostListBottom();
```

Comment:
Kind of a cryptic function name, maybe `jumpPostListToBottom` also isn't called `PostView` now?